src/Storage/StorageFile.php

```diff
@@ -53,9 +54,19 @@ class StorageFile
 		'contentType' => 'text/plain;charset=UTF-8',
 	];
 
-	public function __construct($url, $headers, $bucketId)
+	/**
+	 * StorageFile constructor.
+	 *
+	 * @throws Exception
+	 */
+	public function __construct($bucketId)
```

Comment:
The signature of this class should be:
```
public function __construct($reference_id, $api_key, $bucket_id)
```

Then we should pass in those vars when instantiated. This makes it impossible to use this as a library without the DotEnv library. For testing, we can setup a .env.test that can contain those keys.